src/core/collection/__tests__/Collection.test.tsx

```diff
@@ -59,3 +66,98 @@ describe('event selection module', () => {
     expect(screen.getAllByRole('link').length).toBe(activeEvents.length);
   });
 });
+
+describe('event search module', () => {
+  it.each(
+    [
+      new Date(),
+      subDays(new Date(), 1),
+      subDays(new Date(), 10),
+      subDays(new Date(), 100),
+      subYears(new Date(), 1),
+      subYears(new Date(), 10),
+      subYears(new Date(), 100),
+    ].map((d) => d.toISOString().substring(0, 10)),
```

Comment:
`YYYY-MM-DD'.length` instead of 10 would be more readable and thus more easily graspable at a glance IMHO